content/en/docs/contribute/localization.md

```diff
@@ -175,18 +210,46 @@ For each team, add the list of GitHub users requested in [Add your localization
      - remyleone
 ```
 
+### Open a pull request
+
+Next, [open a pull request](/docs/contribute/new-content/open-a-pr/#open-a-pr) (PR) to add a localization to the `kubernetes/website` repository.
+
+The PR must include all of the [minimum required content](#minimum-required-content) before it can be approved.
+
+For an example of adding a new localization, see the PR to enable [docs in French](https://github.com/kubernetes/website/pull/12548).
+
+### Add a localized README file
+
+To guide other localization contributors, add a new [`README-**.md`](https://help.github.com/articles/about-readmes/) to the top level of k/website, where `**` is the two-letter language code. For example, a German README file would be `README-de.md`.
```

Comment:
nit: k8s/website

Reply:
Convention in Kubernetes contributing is to abbreviate `https://github.com/kubernetes` to `k`. I haven't changed this wording from the existing page, I just moved it. Something for a separate PR?

I do prefer to separate out changes to mechanics (eg the new review mechanism), and keep those apart from general tidying.

Reply:
The wording here is close to the previous version. Also, it's a convention across Kubernetes to abbreviate `https://github.com/kubernetes/` to `k/`.

(we could however make `k/website` a link - maybe in a separate PR - to help people who don't recognize that convention).